src/partial-result-stream.ts

```diff
@@ -379,7 +398,9 @@ export function partialResultStream(
 
   // need types for events-intercept
   // tslint:disable-next-line no-any
-  (requestsStream as any).intercept('error', retry);
+  (requestsStream as any).intercept('error', err => {
```

Comment:
nits: probably better to keep the original one. No need to change the code here.

Reply:
Done.